src/components/clusterWizard/ClusterWizardToolbar.tsx

```diff
@@ -74,7 +74,7 @@ const ValidationSectionToggle: React.FC<ValidationSectionToggleProps> = ({
     return (
       <ToolbarButton variant={ButtonVariant.link} onClick={toggleValidationSection} isInline>
         <WarningTriangleIcon color={warningColor.value} />
-        &nbsp;<small>The configuration form has unresolved validation errors.</small>
+        &nbsp;<small>The form has unresolved validation errors.</small>
```

Comment:
A small change to make the message a bit more universal.

Reply:
Where and when do we show this message?

Reply:
Next to the wizard navigation buttons in case when the step's form has client side validations errors

Reply:
Hi Jiri, Can you please share a screen capture?

Reply:
<img width="1384" alt="Screenshot 2021-04-08 at 16 24 12" src="https://user-images.githubusercontent.com/1121740/114043859-f47ffc00-9886-11eb-818e-24b7da37378a.png">